packages/@azure/servicebus/data-plane/test/stress/fixedNumberOfClientsMultipleQueues.ts

```diff
@@ -0,0 +1,62 @@
+import { ServiceBusClient, SendableMessageInfo } from "../../lib";
+
+const connectionString = "";
+
+const testDurationInMilliseconds = 60000 * 5 * 12 * 24 * 7; // 1 week
+
+const numOfClients = 10;
+
+let msgId = 1;
+
+let snapshotIntervalID: any;
+
+let isJobDone = false;
+
+async function main(): Promise<void> {
+  snapshotIntervalID = setInterval(snapshot, 5000); // Every 5 seconds
+  setTimeout(() => {
+    isJobDone = true;
+  }, testDurationInMilliseconds);
+
+  sendReceiveMessages();
+}
+
+async function sendReceiveMessages(): Promise<void> {
+  const ns = ServiceBusClient.createFromConnectionString(connectionString);
+
+  const clients = [];
+  for (let i = 0; i < numOfClients; i++) {
+    clients[i] = ns.createQueueClient(`t1-queue-new-${i + 1}`);
+  }
+
+  try {
+    while (!isJobDone) {
+      for (let i = 0; i < numOfClients; i++) {
+        const sender = clients[i].createSender();
+        const message: SendableMessageInfo = {
+          messageId: msgId,
+          body: "test",
+          label: `${msgId}`
+        };
+        msgId++;
+        await sender.send(message);
+        const receiver = clients[i].createReceiver();
+        const messagesReceived = await receiver.receiveBatch(1);
+        messagesReceived[0].complete();
```

Comment:
Any reason not to await here? Same in other files

Reply:
Is there a reason why we need to await as such?
I recall discussing on a Teams thread with Mike too about it.
Technically, the program would eventually exit. In this case, I think we particularly don't care about when the success/failure of message completion completes (but we'd likely care if it errors out..).
I see this is tslint's complaint as well - seems like adding .catch() would be better?

Reply:
But since the delay isn't much and doesn't defeat the purpose of test, I'm okay to keep it simple and await on complete.

Reply:
There are 2 aspects to consider here:
- Error thrown from the `complete()
- Ensuring that when the promise returned by `sendReceiveMessages()` resolves, it indicates that we are done with everything that is happening inside

awaiting here ensures that the errors from `complete()` bubbles up gracefully.
Alternative, of course is to add the .catch() and handle the error

To the second point, yes the program eventually will exit.
If we had a console log in`main().then()` that said something like "All done" as an indicator that we are done, then the await here will ensure that we are really done.
If we don't care about that, then we don;t need to await